src/containers/HomePage.js

```diff
@@ -23,7 +23,7 @@ class HomePage extends Component {
             </Link>
           </div>
 
-          <p>*This is a prototype and should not be used to make financial decisions.</p>
+          <p className="home-disclaimer">*This is a prototype and should not be used to make financial decisions.</p>
```

Comment:
Generally speaking, we try to keep each PR to one purpose. This looks good to me, but in future, it'd be helpful to keep this kind of stuff separate.